### PR TITLE
storybook

### PR DESCRIPTION
**Pull request by wassim-arfa**

### CODE REVIEW

File: next-env.d.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```
Comment (arfa) [inline next-env.d.ts:2]: pourquoi ?

**Comment by arfa** [comment]
nice job 🚀

File: components/product/use-price.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  const hasDiscount = baseAmount > amount
+  const formatDiscount = new Intl.NumberFormat(locale, { style: 'percent' })
+  const discount = hasDiscount
+    ? formatDiscount.format((baseAmount - amount) / baseAmount)
+    : null
+
+  const price = formatPrice({ amount, currencyCode, locale })
+  const basePrice = hasDiscount
+    ? formatPrice({ amount: baseAmount, currencyCode, locale })
+    : null
```
Comment (arfa) [inline components/product/use-price.tsx:40]: naming => confusing

https://medium.com/continuousdelivery/naming-one-of-the-hardest-thing-in-software-development-af957d070762

File: next-env.d.ts (hunk 1 above)
Comment (wassim-arfa) [reply next-env.d.ts:2]: after running `npm run dev` the line is auto deleted

**Pull request merged**